src/Workspaces/Core/Portable/Workspace/Solution/ProjectId.cs

```diff
@@ -13,15 +14,18 @@ namespace Microsoft.CodeAnalysis
     /// An identifier that can be used to refer to the same <see cref="Project"/> across versions.
     /// </summary>
     [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
+    [DataContract]
```

Comment:
Given you're still including the custom formatter, did you want to still annotate this like this, or at least leave a comment that a special formatter exists for this one?

Reply:
i'll add a comment.